values.yaml

```diff
@@ -40,12 +40,20 @@ kpi:
       KPI_DEFAULT_FILE_STORAGE: "storages.backends.s3boto3.S3Boto3Storage"
       # -- Set s3 bucket name is relevant
       KPI_AWS_STORAGE_BUCKET_NAME: "kobo-eks-staging"
+      # -- KoBoCAT Django file storage, should match KPI setting
+      KOBOCAT_DEFAULT_FILE_STORAGE: "storages.backends.s3boto3.S3Boto3Storage"
+      # -- KoBoCAT Bucket name 
+      KOBOCAT_AWS_STORAGE_BUCKET_NAME: "kobo-eks-staging"
     secret:
       # Overwrite these
       DATABASE_URL: "postgres://postgres:password@postgres-postgresql:5432/postgres"
       KPI_BROKER_URL: "redis://localhost:6379/1"
       # AWS_ACCESS_KEY_ID: ""
       # AWS_SECRET_ACCESS_KEY: ""
+      # -- Enketo API key. Must match Enketo setting below
+      # ENKETO_API_KEY: ""
```

Comment:
Since it's very important that this var is always the same, I think we should move it to the shared "kobotoolbox". Let me know if you need help on how to reference that in the right Secret objects.

Reply:
Thanks for help. I changed it the way you recommended.